tests/unit_tests/dracut_tests/test_dracut_anaconda-lib.py

```diff
@@ -53,13 +49,19 @@ def _load_script(self, script_name):
         with open(os.path.join("../dracut/", script_name), "rt", encoding="utf-8") as f:
             self._content = f.read()
 
-        self._disable_bash_commands()
-
-    def _disable_bash_commands(self):
+    def _disable_bash_commands(self, disabled_commands):
         disable_list = []
         # disable external and problematic commands in Dracut
-        for disabled_cmd in DISABLE_COMMANDS:
-            disable_list.append(f"""
+        for disabled_cmd in disabled_commands:
+            if isinstance(disabled_cmd, list):
+                disable_list.append(f"""
+{disabled_cmd[0]}() {{
+    echo "{DISABLE_COMMAND_PREFIX}: {disabled_cmd} args: $@" >&2
+    {disabled_cmd[1]}
+}}
+""")
+            if isinstance(disabled_cmd, str):
+                disable_list.append(f"""
```

Comment:
If the appended entry should be separated from existing ones with white space?

Reply:
The first line is empty with `/n` so it is separated by the `f"""` line.

Reply:
Ack